src/dns/index.ts

```diff
@@ -9,9 +15,15 @@ export function lookupTxtRecord(host: string): Promise<string> {
   })
   .then(r => r.json())
   .then(r => r.Answer[0].data)
+  .then(r => r && r.replace(/^"|"$/g, ""))
```

Comment:
I'm notoriously terrible at understanding regex :laughing: 
maybe add a short comment of what this does (I think it strips quotes, right?)

Reply:
Hehe will do. It strips the quotes from the start and end, yeah.